analyses/copy_number_consensus_call_manta/README.md

```diff
@@ -0,0 +1,96 @@
+# Copy Number Consensus Call
+
+## Overview
+
+The PedCan data set contains CNVs called from different callers, ie. Manta, CNVkit, and Freec. 
+The goal is to use all of these callers to reduce false positives and come up with a final consensus list of CNVs.
+This analysis uses information from the following files generated from the 3 callers
+
+* `pbta-cnv-cnvkit.seg.gz`
+* `pbta-cnv-controlfreec.tsv.gz`
+* `pbta-sv-manta.tsv.gz`
```

Comment:
```suggestion
* `cnv-cnvkit.seg.gz`
* `cnv-controlfreec.tsv.gz`
* `sv-manta.tsv.gz`
```

Fingers crossed this works, though i have a vague recollection manta columns changed somewhere and not sure if this was updated, but possibly.